python/ray/train/v2/api/exceptions.py

```diff
@@ -0,0 +1,17 @@
+from typing import Dict
+
+from ray.train.v2._internal.exceptions import RayTrainError
+from ray.util.annotations import DeveloperAPI
+
+
+@DeveloperAPI
+class TrainingFailedError(RayTrainError):
+    """Exception raised by `<Framework>Trainer.fit()` when training fails."""
```

Comment:
Should this be Public(beta)?

Reply:
Let's make it an alpha API for now, we should expose this to users but may want to iterate on the interface a bit more to make it more informative.